ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/parts/PartStack.java

```diff
@@ -145,6 +146,9 @@
    */
   void setDelegate(ActionDelegate delegate);
 
+  /** Returns related to Part Stack properties */
+  PartStackProperties getProperties();
```

Comment:
try to add  Default Implementation here for better backward compatibility

Reply:
@vparfonov 
I have changed implementation so we can avoid extra objects.
Please review.